packages/pluggableWidgets/animation-native/typings/AnimationProps.d.ts

```diff
@@ -8,100 +8,13 @@ import { ActionValue, DynamicValue } from "mendix";
 
 export type AnimationTypeEnum = "in" | "attention" | "out";
 
-export type AnimationInEnum =
-    | "none"
-    | "bounceIn"
-    | "bounceInDown"
-    | "bounceInUp"
-    | "bounceInLeft"
-    | "bounceInRight"
-    | "fadeIn"
-    | "fadeInDown"
-    | "fadeInDownBig"
-    | "fadeInUp"
-    | "fadeInUpBig"
-    | "fadeInLeft"
-    | "fadeInLeftBig"
-    | "fadeInRight"
-    | "fadeInRightBig"
-    | "slideInDown"
-    | "slideInUp"
-    | "slideInLeft"
-    | "slideInRight"
-    | "zoomIn"
-    | "zoomInDown"
-    | "zoomInUp"
-    | "zoomInLeft"
-    | "zoomInRight";
+export type AnimationInEnum = "none" | "bounceIn" | "bounceInDown" | "bounceInUp" | "bounceInLeft" | "bounceInRight" | "fadeIn" | "fadeInDown" | "fadeInDownBig" | "fadeInUp" | "fadeInUpBig" | "fadeInLeft" | "fadeInLeftBig" | "fadeInRight" | "fadeInRightBig" | "slideInDown" | "slideInUp" | "slideInLeft" | "slideInRight" | "zoomIn" | "zoomInDown" | "zoomInUp" | "zoomInLeft" | "zoomInRight";
```

Comment:
Seems like prettier didnt do a good job here

Reply:
This output is from running `release`. I also ran prettier subsequent to these changes, prettier was happy